tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestResultStruct.cs

```diff
@@ -21,6 +21,11 @@ internal struct TestResultStruct
     /// </summary>
     public UnitTestOutcome Outcome;
 
+    /// <summary>
+    /// Gets the display name
+    /// </summary>
+    public string DisplayName;
```

Comment:
Probably technically `string?` I assume, but would require adding `#nullable enable` etc

Reply:
done in [f4d4b7a](https://github.com/DataDog/dd-trace-dotnet/pull/5381/commits/f4d4b7acfdfab96cfbb9680c8ef1299db40978f7)